### PR TITLE
Use thumbor url for all endpoints

### PR DESCRIPTION
**Pull request by aj-may**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -101,7 +101,10 @@ app.route('/spottings/:id')
     res.json(Object.assign(result, {
       id: result[datastore.KEY].name,
       url: `${process.env.BASE_URL}/spottings/${result[datastore.KEY].name}`,
-      image: `https://storage.googleapis.com/andyspottingmedia/${result.image}`,
+      image: thumborURL.setImagePath(`https://storage.googleapis.com/andyspottingmedia/${result.image}`)
+          .resize(100, 100)
```
Comment (marksost) [inline src/index.js]: Should we do 200x200 to account for the high-def screenzzz?

Comment (terowz) [inline src/index.js]: 👍  for Wark's comment. Retna and shiate

Comment (aj-may) [inline src/index.js]: idk how to do that.  @marksost plz advise.

Comment (marksost) [inline src/index.js]: `.resize(200, 200)`

Comment (aj-may) [inline src/index.js]: @marksost not what I meant.  Do i need to provide 2 cuts? one for retna and one for pupil? or just one? what do I do on the frontend? what kind of slick css/js magic do i need to make it work?

**Comment by marksost** [comment]
The front end should "Just work"

https://github.com/andy-spotting/www/blob/master/assets/js/src/andyspotting.js#L97

**Comment by marksost** [comment]
So, just one

**Pull request merged**